action.php

```diff
@@ -295,12 +297,21 @@ public function handleActPreprocessTranslate($event, $param) {
             return;
         }
 
+        // Use suggestion when not specified.
+        if ($use_custom_id) {
+            if (empty($target_id)) {
+                // Not filled. Show form.
+                return;
+            }
+        } else {
+            $target_id = $my->suggestTranslationId($target_title, $target_lang, $source_lang);
```

Comment:
What happens if you user writes an target_id with spaces and other bad characters? I think we need to check that and refuse to save it or at least clean it using the dokuwiki function `cleanID()`.

Also, do you want the user to set the full page id or just the last part after the namespaces? If we want the latter, it is better to use `suggestTranslationId()` on the target_id we got from the browser. It adds the language code as a namespace if that is enabled in the plugin settings and additionally it does `cleanID()` internally so we don't have to worry about bad characters in the target_id we get from the browser. I think this would be better. What do you think?